core/store/migrations/0051_not_null_job_pipeline_spec_id.go

```diff
@@ -0,0 +1,25 @@
+package migrations
+
+import (
+	"gorm.io/gorm"
+)
+
+const up51 = `
+ALTER TABLE jobs ALTER COLUMN pipeline_spec_id SET NOT NULL;
```

Comment:
Won't this break if there's jobs present currently without such an id?

Reply:
It shouldn't be possible - the model doesn't even allow nil values here